public/views/projects-source.html

```diff
@@ -3,6 +3,12 @@
 <div ng-if="!sourceProjects.length && !loadingProjects">
 	There are no exports in progress at this time.
 </div>
+<div>
+	<small ng-show="!boxAuthorized"><strong><a class="btn btn-default btn-xs" href="" data-toggle="modal" data-target="#boxAuthModal">Authorize App to Box</a></strong>
+	</small>
+	<small ng-show="boxAuthorized"><strong><a class="btn btn-default btn-xs" href="" data-toggle="modal" data-target="#boxLogoutModal">Unauthorize App from Box</a></strong>
+	</small>
+</div>
```

Comment:
If boxAuthorized is in the rootScope we can put this anywhere, I think. Let's just go ahead and keep it like this for now.